cvat/apps/engine/signals.py

```diff
@@ -30,18 +26,45 @@ def update_task_status(instance, **kwargs):
         db_task.status = status
         db_task.save()
 
-@receiver(post_save, sender=User, dispatch_uid="create_a_profile_on_create_a_user")
-def create_profile(instance, **kwargs):
+@receiver(post_save, sender=User,
+    dispatch_uid=__name__ + ".create_profile_handler")
+def __create_profile_handler(instance, **kwargs):
     if not hasattr(instance, 'profile'):
         profile = Profile()
         profile.user = instance
         profile.save()
 
-@receiver(post_delete, sender=Task, dispatch_uid="delete_task_files_on_delete_task")
-def delete_task_files_on_delete_task(instance, **kwargs):
+@receiver(post_delete, sender=Project,
+    dispatch_uid=__name__ + ".delete_project_handler")
+def __delete_project_handler(instance, **kwargs):
```

Comment:
These handlers are named inconsistently:

* `__delete_project_handler` - handles project deletion.
* `__create_profile_handler` - handles _user_ creation by creating a profile.

They should all be named either for what they handle or for what they do.

Reply:
renamed to `__save_job_handler`